src/main/java/org/apache/commons/text/WordUtils.java

```diff
@@ -614,13 +615,11 @@ public static String swapCase(final String str) {
         for (int index = 0; index < strLen;) {
             final int oldCodepoint = str.codePointAt(index);
             final int newCodePoint;
-            if (Character.isUpperCase(oldCodepoint)) {
+            if (Character.isUpperCase(oldCodepoint) || Character.isTitleCase(oldCodepoint) ) {
                 newCodePoint = Character.toLowerCase(oldCodepoint);
                 whitespace = false;
-            } else if (Character.isTitleCase(oldCodepoint)) {
-                newCodePoint = Character.toLowerCase(oldCodepoint);
-                whitespace = false;
-            } else if (Character.isLowerCase(oldCodepoint)) {
+            }
+            else if (Character.isLowerCase(oldCodepoint)) {
```

Comment:
Any reason for breaking the line here? Not sure if it's just GitHub UI getting confused, or if the bracket is not in the same line as the else if statement...

Reply:
No, you're right.
My mistake.

Corrected and commited the correction.